airbyte-server/src/main/java/io/airbyte/server/apis/AttemptApiController.java

```diff
@@ -36,6 +41,7 @@ public InternalOperationResult saveStats(final SaveStatsRequestBody requestBody)
   @Override
```

Comment:
Should there be some authorization on the saveStats as well?

Reply:
Maybe.  This endpoint is unimplemented in Cloud and therefore there is nothing to port over.  If we can come up with what we want on those unimplemented endpoints, we can set it.  However, for the sake of porting this over, my plan is to match exactly what is in Cloud at the moment.